core/controller/src/main/scala/whisk/core/controller/Packages.scala

```diff
@@ -76,25 +76,30 @@ trait WhiskPackagesApi extends WhiskCollectionAPI with ReferencedEntities {
    */
   override def create(user: Identity, entityName: FullyQualifiedEntityName)(implicit transid: TransactionId) = {
     parameter('overwrite ? false) { overwrite =>
-      entity(as[WhiskPackagePut]) { content =>
-        val request = content.resolve(entityName.namespace)
+      if (!overwrite && entityName.name.toString == "default") {
```

Comment:
iirc there’s an asString method which is preferred.

Reply:
I’d move the string default to a const on the whisk package Singleton.

Reply:
:+1:  at the same time, i'll make it an array of reserved names that can be expanded as needed